corehq/apps/reports/v2/filters/case_report.py

```diff
@@ -36,8 +36,8 @@ def get_filtered_query(self, query):
         if self.request.can_access_all_locations and EMWF.show_deactivated_data(self.value):
             return query_deactivated_data(query, self.domain)
 
-        # otherwise only return explicit matches
-        case_owners = get_case_owners(self.request, self.domain, self.value)
```

Comment:
Huh, so this was just 500ing before? Or causing a JS error?

Reply:
500ing